enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/ettersending/EttersendelseDto.kt

```diff
@@ -0,0 +1,25 @@
+package no.nav.familie.kontrakter.ef.ettersending
+
+import no.nav.familie.kontrakter.ef.felles.StønadType
+import no.nav.familie.kontrakter.ef.søknad.Vedlegg
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+data class EttersendelseDto(val dokumentasjonsbehov: List<Dokumentasjonsbehov>,
+                            val fnr: String
```

Comment:
Vi bruker `personIdent` på andre steder :) 
```suggestion
                            val personIdent: String
```

Reply:
Meeen, trenger vi egentlige å sende inn/returnere fnr? 
Er det tilsrekkelig å bare sende inn dokumentasjonsbehov?

Reply:
Enig i at vi burde kalle den personIdent. 
Den trengs i mottak for å lagre ettersendinger som er uavhengig søknader. Tror vi beholder det i kontrakten som sådan, men renamer den